contribute/contribute.rst

```diff
@@ -452,38 +462,25 @@ And so are accessible via commands such as:
     journalctl --no-pager  # to avoid line truncation.
 
 When making frontend changes, "Developer Tools" in Chrome/Firefox are critically important.
-You can `inspect elements <https://developer.chrome.com/docs/devtools/dom/>`_ for html/css changes.
-Log to the browser console from javascript code with console.log().
-And use the debugger to step through javascript; all from your browser.
+You can `inspect elements <https://developer.chrome.com/docs/devtools/dom/>`_ for HTML/CSS changes.
+Log to the browser console from your JavaScript code with :code:`console.log()` statements,
+and use the debugger to step through JavaScript; all from your browser.
 
 Adding third party Javascript libraries
 ---------------------------------------
 
-The frontend code uses third party javascript libraries such as jquery,bootstrap, d3 and many others.
+The frontend code uses third party JavaScript libraries such as jQuery, Bootstrap, D3.js and many others.
 These are not part of the rockstor-core repository but are dynamically generated during the build.
 They are placed in the below directory on your build VM:
```

Comment:
```suggestion
They are unzipped into the below directory on your build VM by the build script (build.sh):
```
Just a minor clarification that should help folks to follow what is going on regarding the origin of these files.
This explanation will also work for both rpm in-situ rebuilds and fresh builds such as we have here.